module-utils/unicode/utf8/test/unittest_utf8.cpp

```diff
@@ -356,6 +356,13 @@ TEST_CASE("UTF8: Not ASCII combination")
         REQUIRE_FALSE(combination.toASCII().has_value());
     }
 
+    SECTION("Carier info number - 3 digits")
+    {
+        UTF8 combination = "100";
```

Comment:
please add tests for at least: 
for negative cases:
- 1111  
- 11111 
- 111111
positive cases
- 11147
- 8411
- 11182115

Reply:
Added representative TCs